core/arch/arm/kernel/virtualization.c

```diff
@@ -304,6 +304,9 @@ TEE_Result virt_guest_created(uint16_t guest_id)
 	TEE_Result res = TEE_SUCCESS;
 	uint32_t exceptions = 0;
 
+	if (guest_id == HYP_CLNT_ID)
+		return OPTEE_SMC_RETURN_ENOTAVAIL;
```

Comment:
Should return a `TEE_ERROR_xxx` value. Maybe `TEE_ERROR_BAD_PARAMETERS`.

Reply:
Noted.